Source/WebCore/page/Page.h

```diff
@@ -1079,28 +1086,28 @@ class Page : public Supplementable<Page>, public CanMakeWeakPtr<Page> {
     void updateElementsWithTextRecognitionResults();
 #endif
 
-    const std::unique_ptr<Chrome> m_chrome;
-    const std::unique_ptr<DragCaretController> m_dragCaretController;
+    UniqueRef<Chrome> m_chrome;
```

Comment:
I'm a little sad that these can't stay const, indicating that the object can't be changed, while keeping the const correct model of UniqueRef, which only allows you to get a non-const object reference from it if it is not const.  I think this is still an improvement, though.

Nit: T* ptr() const in UniqueRef.h should be const T* ptr() const

Reply:
right, let's do that in a follow up